Mathlib/Analysis/Convex/Normed.lean

```diff
@@ -133,3 +135,58 @@ theorem isConnected_setOf_sameRay_and_ne_zero {x : E} (hx : x ≠ 0) :
     IsConnected { y | SameRay ℝ x y ∧ y ≠ 0 } := by
   simp_rw [← exists_pos_left_iff_sameRay_and_ne_zero hx]
   exact isConnected_Ioi.image _ (continuous_id.smul continuous_const).continuousOn
+
+end SeminormedAddCommGroup
+
+section NormedAddCommGroup
+variable [NormedAddCommGroup E] [NormedSpace ℝ E] [FiniteDimensional ℝ E] {s : Set E} {x : E}
+
+/-- We can intercalate a simplex between a point and one of its neighborhoods. -/
+lemma exists_mem_interior_convexHull_affineBasis (hs : s ∈ 𝓝 x) :
+    ∃ b : AffineBasis (Fin (finrank ℝ E + 1)) ℝ E,
+      x ∈ interior (convexHull ℝ (range b)) ∧ convexHull ℝ (range b) ⊆ s := by
```

Comment:
Is there some stronger statement along the lines of "and the points of the simplex lie on the `interior` or `frontier` of `s`"?

Reply:
Because of [`interior_mem_nhds`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Basic.html#interior_mem_nhds), this lemma is easily strengthenable to `...  ∧ convexHull ℝ (range b) ⊆ interior s`. I don't think I should be making that a lemma though as the current is syntactically more general and the stronger one is easily derived